### PR TITLE
Minor refactoring in Redis connector

### PR DESCRIPTION
**Pull request by polaris6**

### CODE REVIEW

File: plugin/trino-redis/src/test/java/io/trino/plugin/redis/TestRedisConnectorTest.java [hunk 1]
```diff
@@ -46,30 +46,14 @@ protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
     {
         switch (connectorBehavior) {
             case SUPPORTS_TOPN_PUSHDOWN:
-                return false;
```
[the remark this answers is not in the document]
Comment (polaris6) [reply plugin/trino-redis/src/test/java/io/trino/plugin/redis/TestRedisConnectorTest.java]: Thank you for your review, I removed the changes to this part.

File: plugin/trino-redis/src/main/java/io/trino/plugin/redis/RedisDataType.java [hunk 2]
```diff
@@ -18,5 +18,5 @@
  */
 public enum RedisDataType
 {
-    STRING, HASH, SET, ZSET
```
[the remark this answers is not in the document]
Comment (polaris6) [reply plugin/trino-redis/src/main/java/io/trino/plugin/redis/RedisDataType.java:21]: Thank you for your review, `SET` is not used, so I removed it temporarily, do you think it's okay to do this?

**Comment by polaris6** [comment]
@ebyhr Hi ebyhr, I submitted a cla yesterday, but I haven't received a reply yet, so the current pr submission may fail the verification, which can be ignored for now. Thank you very much!
企业微信截图_3fef04b7-01da-4025-895d-6cd7af5294f3

File: plugin/trino-redis/src/main/java/io/trino/plugin/redis/RedisDataType.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (polaris6) [reply plugin/trino-redis/src/main/java/io/trino/plugin/redis/RedisDataType.java:21]: There is no such plan yet, I plan to submit a pr later to improve the query performance of redis's `STRING` and `HASH` value types.

[the remark this answers is not in the document]
Comment (ebyhr) [reply plugin/trino-redis/src/main/java/io/trino/plugin/redis/RedisDataType.java:21]: Cool, please extract a commit for this change and clarify that it's unused in the commit body.

**Review by ebyhr — APPROVED** [review]
Please squash commits and improve the commit message:
```
Minor improvement to RedisRecordCursor
```

https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#format-git-commit-messages

**Comment by polaris6** [comment]
@ebyhr Thank you for your suggestion, I resubmitted it.

**Comment by ebyhr** [comment]
@polaris6 Contributors list was updated today, but your account doesn't exist. Just in case, did you send CLA with `@polaris6` account name?

**Comment by polaris6** [comment]
Hi @ebyhr, the account name is correct, but there is some problem with the signature, I have re-sent the email to Martin, thank you very much for the reminder.

**Comment by martint** [comment]
@cla-bot check

**Comment by ebyhr** [comment]
Thanks!

**Pull request merged**